hack/upgrade-test-harness/conf.yaml

```diff
@@ -42,5 +42,5 @@ testParams:
     operatorVersion: 2.12.1
     stackVersion: 8.13.2
   - name: upcoming
-    operatorVersion: 2.13.0-SNAPSHOT
-    stackVersion: 8.14.0-SNAPSHOT
+    operatorVersion: 2.14.0-SNAPSHOT
+    stackVersion: 8.15.0-SNAPSHOT
```

Comment:
You probably want to make these change in a separate PR. Check https://github.com/elastic/cloud-on-k8s/commit/0b9cf09b091d822d045e7be22faeda7ceeebbd32

Reply:
@naemono so this must be done after the release ?

Reply:
No, there are just additional changes that go along with it. You can add them to this PR if you'd like, but if you just remove this, it's good to go. The upgrade-test-harness pieces are a bit more involved.

Reply:
aah got it, will open a separate PR for that then.